app/controllers/discourse_code_review/code_review_controller.rb

```diff
@@ -95,6 +95,39 @@ def followup
       render_next_topic(topic.category_id)
     end
 
+    def followedup
```

Comment:
You are probably going to hate me for this suggestion, but `followed_up` is two words and should have an underscore.

*ducks*

Reply:
I had doubts when I was writing it, but decided to go with "followedup" just to be similar with "followup".